<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsSearch</class>
 <widget class="QWidget" name="InsSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Идентификатор: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="idEdit">
       <property name="text">
        <string>=0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Имя: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="nameEdit">
       <property name="text">
        <string>=0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Средняя частота: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="freqEdit">
       <property name="text">
        <string>=0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Класс:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Не важно</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Барабан</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Тарелки</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Скрипка</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Электрогитара</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Синтезатор</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Орган</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QRadioButton" name="groupSearchButton">
       <property name="text">
        <string>По свойствам групп</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="groupAdd"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
